boxsdk/session/session.py

```diff
@@ -242,8 +249,11 @@ def with_default_network_request_kwargs(self, extra_network_parameters):
         kwargs['default_network_request_kwargs'].update(extra_network_parameters)
         return self.__class__(**kwargs)
 
+    # pylint: disable=unused-argument
     def _get_retry_after_time(self, attempt_number, retry_after_header):
         """
+        We are using the attempt number that failed to calculate the retry time for the next retry attempt.
```

Comment:
This comment should probably be integrated with the broader description of this method, maybe something like this:

`Get the amount of time to wait before retrying the API request, using the attempt number that failed to calculate the retry time for the next retry attempt.`